### PR TITLE
Feat/story-133 home page

### PR DESCRIPTION
**Pull request by mat10tng**
origin/master
home
this change will add

1. splash screen just for the homepage
2. modified change to the withLoading Hoc to accept splash screen
3.New Home page

splash
new home

### CODE REVIEW

File: src/app/page/Home/Home.js [hunk 1]
```diff
@@ -4,62 +4,141 @@ import PropTypes from "prop-types";
...
-          <StyledHomeButton
-            onClick={() => history.push("/request/create")}
-            text="Request Help"
-            secondary
-          />
-          <StyledHomeButton onClick={() => history.push("/about")}>About</StyledHomeButton>
-        </>
+        <Grid container>
+          <Grid container>
+            <img src={HomeImage1} className={classes.HomeImage} alt="" />
```
Comment (azmainamin) [inline src/app/page/Home/Home.js:85]: We should add some alt tag to make aXe stop yelling.

File: src/app/page/Home/Home.js [hunk 2]
```diff
@@ -4,62 +4,141 @@ import PropTypes from "prop-types";
...
+          </Grid>
+          <Grid container>
+            <Button className={classes.LoginButton} onClick={() => history.push("/login")}>
+              Log in
```
Comment (azmainamin) [inline src/app/page/Home/Home.js:89]: Can we rename to Signin instead of login?

File: src/app/page/Home/Home.js (hunk 1 above)
Comment (mat10tng) [reply src/app/page/Home/Home.js:85]: this one is decorative and so have a alt=""  (empty string)

File: src/app/page/Home/Home.js (hunk 2 above)
Comment (mat10tng) [reply src/app/page/Home/Home.js]: yeah

**Pull request merged**